test/sharness/t0250-files-api.sh

```diff
@@ -296,6 +296,10 @@ test_files_api() {
     ipfs files cp /ipfs/$FILE3 /cats/this/is/a/dir/file3
   '
 
+  test_expect_success "can copy file into deep dir using -p flag $EXTRA" '
```

Comment:
Can we make sure the file was actually created? I.e., try reading it.

(trying to avoid regressions)